### ISSUE TITLE
Online Spreadsheet Viewer of document server interfere with keyboard and javascript event

### ISSUE THREAD

**Issue by nettree** [ONLYOFFICE/DocumentServer#107]
**Do you want to request a *feature* or report a *bug*?**
This is a potential bug, and only occurs in IE11

**What is the current behavior?**
When embed online editor in a iframe in a web page, after an Microsoft Excel document is loaded in editor, some keys in keyboard will never work, which means you can't type in these letters anywhere in web page. These keys are:  b, u, i, y, z, `, 1, 2, 3, 4, 5, 6.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
1. Embed online editor in iframe and put iframe in a web page
2. Load an excel document (sometimes docx document doesn't work as well) in online editor by using IE11
3. After editor is loaded up, typing the keys above within the text field or anywhere you can input information of the page

**What is the expected behavior?**
Keys should be able to be typed even if online editor is loaded

**Which versions of DocumentServer, and which browser / OS are affected by this issue? Did this work in previous versions of DocumentServer?**
This happens in latest version 4.3.2, only reproduce in IE11, it doesn't work in previous versions of DocumentServer.

**Comment by JohnPerkin** [comment]
Hello @nettree!
We tried to reproduce the issue on PC Win 7 Ult SP1 x64 with IE 11.0.9600.18638 update versions 11.0.41 but everything worked well. Please describe the issue more detalied

**Comment by nettree** [comment]
Hi @JohnPerkin,

Thank you for your quick response, below is my config object passed to DocsAPI.DocEditor() method:

```
var config = {
                type: "desktop",
                width: "100%",
                height: "100%",
                documentType: getDocumentType(docType),
                document: {
                    title: docName,
                    url: this.attributes.docUrl,
                    fileType: translateDocumentType(docType),
                    key: this.attributes.key,
                    permissions: {
                        edit: true
                    }
                },
                editorConfig: {
                    mode: "view",
                    callbackUrl: this.attributes.callbackUrl,
                    user: this.attributes.user
                }
            };
```
If I change type of config object to `embedded` from `desktop` then the issue can't be reproduced and everything works well, it suspect that something happened between Spreadsheet Viewer and my web application and those key stroke events were consumed by Spreadsheet Viewer so that they can't be typed in. The reason I'm saying that is because letters `b`, `u`, `i`, `y`, `z` that can't be typed in stand for `bold`, `underline`, `Italic`, `redo`, `undo` if we use `Ctrl` with them in OnlyOffice editor, I'm not sure whether this is coincidence, but hopefully this is a clue for this issue. However, I have no idea why the other keys (` 1 2 3 4 5 6) can't be typed in.

Another issue we found previously in version 4.2.x is that mouse click event of some elements in our web page doesn't work if we have OnlyOffice editor loaded, the behaviour is that when you click those elements, it was supposed to expand or collapse the div, but you can only see the screen scroll up to the top of the page. This can only be reproduced in IE11 as well, however, when we upgraded OnlyOffice document server to 4.3.1, this issue can't be reproduced any more, so I assume that you guys fix this issue in version 4.3.1, I hope this information is helpful for you to identify the current issue raised here because it looks like the similar behaviour.

Anyway, we're going to use type:"embedded" for the time being, thank you for your help and you can close the ticket if you want.
